README.md

```diff
@@ -1,5 +1,14 @@
 # Blockly [![Build Status]( https://travis-ci.org/google/blockly.svg?branch=master)](https://travis-ci.org/google/blockly)
 
+## Quick Usage
+
+```bash
+> n lts
+> npm install
+> npm run build
+> google-chrome tests/rovercode_playground.html
+> # import tests/rovercode-all.xml. Generate javascript and compare to tests/rovercode-all-output.js
```

Comment:
I figured out how to use their unit testing system. It's really cool.
![image](https://user-images.githubusercontent.com/976857/80058062-3852e780-84f6-11ea-8424-7b528ca72bd2.png)
But unfortunately, there's no support for mocking. Our blocks are not very functional, so there wasn't really any way to assert on what any of them returned.

As some assurance that I've tried out these blocks, you can load up the `rovercode-all.xml` program in the playground and assure that its output matches `rovercode-all-output.js`. I can work on some way for the CI to do this.

Reply:
![image](https://user-images.githubusercontent.com/976857/80058573-a350ee00-84f7-11ea-8e70-965b4836a235.png)

```javascript
var a;


setMotor('LEFT', 'FORWARD', '99');
setMotor('LEFT', 'FORWARD', '');
stopMotor('LEFT');
a = (getLightSensorValue('LEFT'));
a = (buttonHasBeenPressed('A'));
displayMessage(String('Sup'));
displayMessage(String(123));
displayMessage(String(true));

```